src/main.rs

```diff
@@ -82,6 +82,9 @@ fn main() {
 
     let net = cmd_arguments.value_of("net");
 
+    // This .unwrap() cannot fail as there is a default value defined
+    let console = cmd_arguments.value_of("console").unwrap();
```

Comment:
I think you're missing adding the argument to `cmd_arguments`

Reply:
I failed to understand what else should I do. Could you please check the latest code to see if it is still necessary?

Reply:
Yes, currently it panics if you don't pass any argument that matches as there is no default value is defined.

Please make console control a boolean and enabled by default. The easiest way to do that is via something like
```
        .arg(
            Arg::with_name("disable-console")
                .long("disable-console")
                .help("Disable (virtio) console")
                .default_value(false),
        )
```

You'll then need to go through the rest of the code and change it so that console control is determined by a boolean. e.g. `ConsoleConfig` etc.

Reply:
And then you can check with `let enable_console = !cmd_arguments.is_present("disable-console");`

Reply:
Implemented this one. Thanks!